addons/point_of_sale/models/pos_config.py

```diff
@@ -355,6 +355,17 @@ def _check_payment_method_receivable_accounts(self):
                 _("You must configure an intermediary account for the payment methods: %s.") % method_names
             )
 
+    @api.constrains('module_pos_loyalty')
```

Comment:
That's not really what I had in mind. you should move the check in the `pos_loyalty` module. Btw, the commit name should be: `[FIX] pos_loyalty: ...`